--- origsrc/gnome-python-desktop-2.22.0/gtop/gtop.c	2008-02-09 10:26:43.000000000 -0600
+++ src/gnome-python-desktop-2.22.0/gtop/gtop.c	2008-06-03 19:07:30.781250000 -0500
@@ -782,7 +782,9 @@
 	glibtop_netload buf;
 	union {
 		char ip4[INET_ADDRSTRLEN];
+#ifdef AF_INET6
 		char ip6[INET6_ADDRSTRLEN];
+#endif
 		char hw [32]; /* 8 * 2 + 7 * 1 ... 32 is enough :) */
 	} addr;
 	const char *iface;
@@ -817,11 +819,13 @@
 	my_dict_add_and_decref(d, "errors_total", PyL_ULL(buf.errors_total));
 	my_dict_add_and_decref(d, "collisions",   PyL_ULL(buf.collisions));
 
+#ifdef AF_INET6
 	my_dict_add_and_decref(d, "prefix6",
 			     PyS_S(inet_ntop(AF_INET6, buf.prefix6, addr.ip6, sizeof addr.ip6)));
 	my_dict_add_and_decref(d, "address6",
 			     PyS_S(inet_ntop(AF_INET6, buf.address6, addr.ip6, sizeof addr.ip6)));
 	my_dict_add_and_decref(d, "scope6", PyI_L(buf.scope6));
+#endif
 
 	my_dict_add_and_decref(d, "hwaddress",
 			     PyS_S(hwaddress_format_for_display(&buf, addr.hw, sizeof addr.hw)));
