--- origsrc/gnome-python-desktop-2.32.0/evolution/ebookmodule.c	2011-12-08 23:20:46.212054400 -0600
+++ src/gnome-python-desktop-2.32.0/evolution/ebookmodule.c	2011-12-08 23:17:26.861431000 -0600
@@ -21,6 +21,7 @@
 #endif
 
 #include <pygobject.h>
+#include <libedataserver/eds-version.h>
 
 void pyebook_register_classes(PyObject *d);
 void pyebook_add_constants(PyObject *m, const gchar *strip_prefix);
@@ -35,9 +36,11 @@ initebook(void)
 
     if (!pygobject_init(-1, -1, -1))
         return;
+#if !EDS_CHECK_VERSION(2, 32, 0)
     PyImport_ImportModule("bonobo");
     if (PyErr_Occurred())
         return;    
+#endif
 	
     m = Py_InitModule("ebook", pyebook_functions);
     d = PyModule_GetDict(m);
--- origsrc/gnome-python-desktop-2.32.0/evolution/ecalmodule.c	2011-12-08 23:20:46.212054400 -0600
+++ src/gnome-python-desktop-2.32.0/evolution/ecalmodule.c	2011-12-08 23:17:30.471436000 -0600
@@ -21,6 +21,7 @@
 #endif
 
 #include <pygobject.h>
+#include <libedataserver/eds-version.h>
 
 void pyecal_register_classes(PyObject *d);
 void pyecal_add_constants(PyObject *m, const gchar *strip_prefix);
@@ -35,9 +36,11 @@ initecal(void)
     /* perform any initialisation required by the library here */
     if (!pygobject_init(-1, -1, -1))
         return;
+#if !EDS_CHECK_VERSION(2, 32, 0)
     PyImport_ImportModule("bonobo");
     if (PyErr_Occurred())
         return;    
+#endif
 	
     m = Py_InitModule("ecal", pyecal_functions);
     d = PyModule_GetDict(m);
